enos/enos-descriptions.hcl

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
I did a search and I still see many files that are BUSL. Can we update all the files?